platform-coop-toolkit/coop-map-directory-index
# feat: update Organization.code_availability to match data

## Description

This PR brings the model for `Organization.code_availability` in line with what's in the database.

## Steps to test

1. Check out this branch.
2. Run the migration.
3. Try to edit an organization in the admin, setting/modifying values for _Code availability_.

**Expected behavior:** All code availability values can be saved to the database.

## Additional information

Not applicable.

## Related issues

Partially addresses #64.
